streaming/base/storage/upload.py

```diff
@@ -855,22 +856,40 @@ def __init__(self,
                  exist_ok: bool = False) -> None:
         super().__init__(out, keep_local, progress_bar, retry, exist_ok)
 
-    def upload_file(self, filename: str):
+    def upload_file(self, filename: str, timeout: int = 10):
         """Upload file from local instance to Databricks Unity Catalog.
 
         Args:
             filename (str): Relative filepath to copy.
+            timeout (int): Seconds for each try to upload.
         """
 
+        def wait_for_file(remote_filename: str, file_size: int):
+            """Wait for a file to be fully available."""
+            start_time = time.time()
+            while time.time() - start_time < timeout:
+                try:
+                    file_info = self.client.files.get_status(remote_filename)
+                    if abs(file_info.file_size - file_size) < 10:
+                        return True
+                except Exception as _:
```

Comment:
```suggestion
                except:
```
since you don't do anything with the exception